CHANGELOG.md

```diff
@@ -1,9 +1,20 @@
 # React Native Module Changelog
 
+## Version 15.2.5 - May 3, 2023
+
+Patch release that fixes a issue with modifying attributes on iOS silently fail. Apps that are using
```

Comment:
should this be `... silently failing.`, or maybe `... attributes on iOS that could silently fail.`?